InnerEye/ML/Histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -54,17 +54,26 @@ def __init__(self,
                  transform: Optional[Callable] = None,
                  **kwargs: Any) -> None:
         super().__init__(root=root, transform=transform)
-        self.base_dataset = PandaTilesDataset(root=root,
+        # filter out tiles based on occupancy threshold
+        self.full_dataset = PandaTilesDataset(root=root,
                                               dataset_csv=dataset_csv,
                                               dataset_df=dataset_df)
+        self.occupancy_threshold = 0
```

Comment:
shouldn't we set this parameter as parameter of the constructor? so that it becomes something we can configure from the container? otherwise to run again this experiment we will need to change each time this class

Reply:
@vale-salvatelli The issue is that there's no clean way to pass arguments into this SSL dataset, because it's instantiated from by the base SimCLR container. A workaround might be to declare `occupancy_threshold` as a class attribute and overwrite it from our container.

Reply:
I am in favor of this solution

Reply:
I have now changed this as suggested